### PR TITLE
Quick Start / Getting Started Page

### PR DESCRIPTION
**Pull request by codeithuman**
## Please verify the following:
- [x] Everything works on iOS/Android
- [x] `ignite-base` **ava** tests pass
- [x] `fireDrill.sh` passed

## Describe your PR

TODO: 
- [x] Finish `Testing your Ignite Installation` section
- [x] Fix `ignite` npm package name, should be `react-native-ignite`

This PR will add `/docs/quick-start/getting-started.md`. This page describes how to get started with Ignite 2 by downloading it from NPM.

The directory structure is based off other Facebook Open Source projects, such as Flow, Nuclide and ReactJS. These projects have a "Quick Start" navigation heading (in the sidebar) with a "Getting Started" page/sub-link. 

For example: 

screen shot 2017-02-15 at 11 07 41 am
screen shot 2017-02-15 at 11 07 53 am
screen shot 2017-02-15 at 11 07 58 am

### CODE REVIEW

File: docs/quick-start/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Welcome to Ignite, the ideal starting app for React Native, best practices, generators, and more. This page will walk you through installing Ignite and using Ignite to generate a React Native Project.
+
+To use Ignite, you will need to install React Native. The best way to install React Native is to follow the React Native installation instructions found on official React Native website, [here](http://facebook.github.io/react-native/docs/getting-started.html). These instructions will walk your through installing React Native on your specific platform.
+
+## Installing Ignite Dependencies
+
+Installing Ignite is easy with one simple terminal command:
+
+```
+npm install -g ignite
```
Comment (skellock) [inline docs/quick-start/getting-started.md]: It's `react-native-ignite`. We have the wrong package name in the source code! Ha!  Il. Fix that.

**Comment by mlaco** [comment]
Looks good to me aside from the items already mentioned.

File: packages/ignite/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ignite",
+  "name": "react-native-ignite",
```
Comment (codeithuman) [inline packages/ignite/package.json]: @skellock Is this where you were talking about updating the NPM name?

Comment (skellock) [inline packages/ignite/package.json]: Yes, there's also a devDependency located in the `ignite-integration-testing` `package.json` too.

**Comment by jamonholmgren** [comment]
Great job @codeithuman !

**Pull request merged**